include/kodlab_mjbots_sdk/robot_base.h

```diff
@@ -171,5 +181,6 @@ namespace kodlab
         real_time_tools::Timer run_timer_;                            /// Run timer for robot, started at construction
         SoftStart soft_start_;                                        /// Soft Start object
         int num_joints_ = 0;                                          /// Number of joints
+        std::unordered_map<std::string, int> joint_name_to_index_;    /// Map from joint name to joint index
```

Comment:
For the [Doxygen to work properly](https://doxygen.nl/manual/docblocks.html):
```suggestion
        std::unordered_map<std::string, int> joint_name_to_index_;    ///< Map from joint name to joint index
```